profiles/api/errors.py

```diff
@@ -0,0 +1,52 @@
+class OAuth2Error(Exception):
```

Comment:
I don't think there's anything useful to extend as a base exception class in https://docs.python.org/3/library/exceptions.html but in other cases there could be

Reply:
I had wondered about `HTTPException`, but didn't feel right.